### PR TITLE
Map Page: change province list bottom sheet to react-picker, add myLocation button

### PR DESCRIPTION
**Pull request by codingki**

### CODE REVIEW

**Comment by agallio** [comment]
Please remove the `package-lock.json`.

**Review by agallio — CHANGES_REQUESTED** [review]
Please remove the `package-lock.json`

**Comment by codingki** [comment]
ok!

**Comment by agallio** [comment]
I think it's better to move the get location button to bottom when responsive, @codingki

Screen Shot 2021-07-08 at 20 27 43

File: pages/map.js [hunk 1]
```diff
@@ -11,8 +11,9 @@ import mapboxgl from '!mapbox-gl'
 import styles from '@/styles/Map.module.css'
 
 import useHospitalDataByProvince from '@/hooks/useHospitalDataByProvince'
-
+import Select from 'react-select'
```
Comment (kelilipan) [inline pages/map.js:14]: i think we should re-use "search" component from main page to reduce bundle size

Comment (codingki) [reply pages/map.js:14]: good idea, but I think better change this after we implement Chakra UI in map page

**Comment by codingki** [comment]
> I think it's better to move the get location button to bottom when responsive, @codingki
> 
> Screen Shot 2021-07-08 at 20 27 43

good catch, didn't see that before

**Comment by agallio** [comment]
Could you give more padding? Like 10-20 px above the 'Jumlah Rumah Sakit' card

localhost_3000_map(iPhone X)

**Comment by codingki** [comment]
it should be good now

**Comment by agallio** [comment]
🆗 Lemme check

**Comment by codingki** [comment]
where do you deploy this to? vercel right? why there's no preview. I think you should integrated your github to your vercel account

**Comment by agallio** [comment]
My account has reached the git integrations limit. 😔
So I do all the work manually wkwkwkwk

**Review by agallio — APPROVED** [review]
Looks great! Brace for impact #18 @svspicious. I'll ship this first. 🚢

**Comment by agallio** [comment]
Sorry for merging this by myself. 🙇🏻

**Pull request merged**